component/Review/formhelper/FormSlideComponent.jsx

```diff
@@ -76,229 +81,260 @@ export function FormSlideComponent({
-                  handleAnswerChange(id, e.target.value, "input_box");
+              <span className="absolute top-2 lg:top-3 right-0  p-2 lg:p-3 leading-0 cursor-pointer rounded-full hover:bg-gray-100">
+                <CloseOutlined />
+              </span>
+            </Popconfirm>
+          </p>
 
-                  answerHandle(id, e.target.value);
-                }}
-                className="w-full h-full "
-                required={true}
-                size="large"
+          <div className="pb-4 lg:pb-6 space-y-4 lg:space-y-8">
+            <p className="text-lg  lg:text-2xl font-bold text-primary">
+              {questionText}
+            </p>
+            {options?.length > 0 && type === "checkbox" && (
+              <Form.Item
+                name={"ques" + id}
+                rules={[
+                  {
+                    required: true,
+                    message: "",
+                  },
+                ]}
+                className="w-full h-full lg:max-w-2xl mx-auto"
               >
-                <Row gutter={[32, 32]} justify="center">
-                  {options?.map((op, j) => {
-                    return (
-                      <Col xs={24} md={12} lg={8} key={j + "option"}>
-                        <Radio.Button
-                          className="text-center answer-radio-button w-full flex items-center justify-center p-2 lg:p-3 rounded-md"
-                          value={op.optionText}
-                        >
-                          {op.optionText}
-                        </Radio.Button>
-                      </Col>
-                    );
-                  })}
-                </Row>
-              </Radio.Group>
-            </Form.Item>
-          )}
-          {type == "input" && (
-            <div>
-              <div className="text-right  text-xs md:text-base text-primary mb-1 ">
-                Text Limit :{" "}
-                <span className=" font-semibold ">{inputLimit[id] ?? 180}</span>
-              </div>
-              <Form.Item name={"ques" + id}>
-                <Input
-                  size="large"
-                  placeholder={type == "input" ? "Short Text" : ""}
+                <Radio.Group
                   onChange={(e) => {
+                    handleAnswerChange(id, e.target.value, "input_box");
+
                     answerHandle(id, e.target.value);
-                    handleAnswerChange(id, e.target.value);
-                    handleInputLimit(id, e.target.value);
                   }}
-                  className="rounded-md"
-                  maxLength={180}
-                />
+                  className="w-full h-full "
+                  required={true}
+                  size="large"
+                >
+                  <Row gutter={[32, 32]} justify="center">
+                    {options?.map((op, j) => {
+                      return (
+                        <Col xs={24} md={12} lg={8} key={j + "option"}>
```

Comment:
Why you are using index for the key , you can use any attribute that is different for all